doc/Programming-guide/Using-Kaa-endpoint-SDKs/Java/index.md

```diff
@@ -3,16 +3,201 @@ layout: page
 title: Java
 permalink: /:path/
 sort_idx: 40
+
 ---
 **Table of Contents**
 
 - [Minimum Requirements](#minimum-requirements)
 - [Using endpoint SDK in your appication](#using-endpoint-sdk-in-your-appication)
 
```

Comment:
Add a few sentences introducing reader to the Java SDK, explaining when to prefer it to the other SDKs, etc.